### ISSUE TITLE
Missing SemiBold and SemiBoldItalic

### ISSUE THREAD

**Issue by m4rc1e** [googlefonts/opensans#2]
All styles which are currently being [served](https://fonts.google.com/specimen/Open+Sans) must exist

![screen shot 2018-02-07 at 08 54 15](https://user-images.githubusercontent.com/7525512/35907000-7f04b20e-0be4-11e8-9965-e9bb5a046e49.png)

**Comment by laerm0** [comment]
Ah, so the issue is that the names of Semi and Extra just need to be changed to have the hyphens, right?

**Comment by m4rc1e** [comment]
Nope,

you're missing the weights. The [fonts dir](https://github.com/googlefonts/opensans/tree/master/ttfs) in this repo does not contain Semi weights.

As a side note, GF displays SemiBold as Semi-Bold on the site. This happens on the front end only, files are not touched.

**Comment by laerm0** [comment]
Oh, that's old data. My new stuff [here](https://github.com/laerm0/opensans) hasn't been PR'd in yet. Check those files please.

**Comment by m4rc1e** [comment]
Compile and ship them please. The ttf still doesn't have those weights.

**Comment by laerm0** [comment]
I'm confused... Open Sans wasn't manually hinted. Also, the `hinted_ttfs` folder clearly has semibold. The `ttfs` directory was from last year and all that was was my merging Hebrew into the Latin.

**Comment by m4rc1e** [comment]
Thanks, much clearer.

What's the point of the ttfs dir. Sounds like these fonts are out of date?

**Comment by laerm0** [comment]
Yeah, I guess those were a previous step in the process. It didn't occur to me that you'd be looking at the ttfs and not the Glyphs source. I suppose all GF technically cares about is how the ttfs look, right?

**Comment by davelab6** [comment]
How was it hinted?

**Comment by laerm0** [comment]
`ttfautohint --composites --strong-stem-width GD` right out of Glyphs.
